rpm/build/SPECS/jenkins.spec

```diff
@@ -39,15 +41,15 @@ Authors:
 
 %install
 rm -rf "%{buildroot}"
-%__install -D -m0644 "%{SOURCE0}" "%{buildroot}%{_prefix}/%{name}.war"
+%__install -D -m0644 "%{SOURCE0}" "%{buildroot}%{_javadir}/%{name}.war"
 %__install -d "%{buildroot}%{workdir}"
 %__install -d "%{buildroot}%{workdir}/plugins"
 
 %__install -d "%{buildroot}/var/log/%{name}"
 %__install -d "%{buildroot}/var/cache/%{name}"
 
 %__install -D -m0755 "%{SOURCE1}" "%{buildroot}/etc/init.d/%{name}"
-%__sed -i 's,~~WAR~~,%{_prefix}/%{name}.war,g' "%{buildroot}/etc/init.d/%{name}"
+%__sed -i 's,~~WAR~~,%{_javadir}/%{name}.war,g' "%{buildroot}/etc/init.d/%{name}"
```

Comment:
Is the whole `init.d` part still relevant? Can't you just drop it?

Reply:
Can be done in a subsequent cleanup PR once we're sure they're not needed. You are welcome to submit that PR.